Subject: ctl-472

---
 drivers/hid/wacom_wac.c | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/drivers/hid/wacom_wac.c b/drivers/hid/wacom_wac.c
index aa692e28b2cd..920f3d6bbc71 100644
--- a/drivers/hid/wacom_wac.c
+++ b/drivers/hid/wacom_wac.c
@@ -4376,6 +4376,12 @@ static const struct wacom_features wacom_features_0x360 =
 static const struct wacom_features wacom_features_0x361 =
      { "Wacom Intuos Pro L", 62200, 43200, 8191, 63,
        INTUOSP2_BT, WACOM_INTUOS3_RES, WACOM_INTUOS3_RES, 9, .touch_max = 10 };
+static const struct wacom_features wacom_features_0x37A =
+     { "One by Wacom CTL472", 14720, 9225, 2047, 63,

+       BAMBOO_PEN, WACOM_INTUOS_RES, WACOM_INTUOS_RES };
+static const struct wacom_features wacom_features_0x37B =
+     { "One by Wacom CTL672", 21648, 13530, 2047, 63,

+       BAMBOO_PEN, WACOM_INTUOS_RES, WACOM_INTUOS_RES };

 static const struct wacom_features wacom_features_HID_ANY_ID =
      { "Wacom HID", .type = HID_GENERIC, .oVid = HID_ANY_ID, .oPid = HID_ANY_ID };
@@ -4544,6 +4550,8 @@ const struct hid_device_id wacom_ids[] = {
      { USB_DEVICE_WACOM(0x343) },
      { BT_DEVICE_WACOM(0x360) },
      { BT_DEVICE_WACOM(0x361) },
+     { USB_DEVICE_WACOM(0x37A) },
+     { USB_DEVICE_WACOM(0x37B) },
      { USB_DEVICE_WACOM(0x4001) },
      { USB_DEVICE_WACOM(0x4004) },
      { USB_DEVICE_WACOM(0x5000) },
--
2.11.0
